wts/api.py

```diff
@@ -133,10 +135,12 @@ def _setup(app):
         "Aggregate endpoint allowlist: %s", app.config["AGGREGATE_ENDPOINT_ALLOWLIST"]
     )
     db.init_app(app)
-    app.register_blueprint(oauth2.blueprint, url_prefix="/oauth2")
-    app.register_blueprint(tokens.blueprint, url_prefix="/token")
-    app.register_blueprint(external_oidc.blueprint, url_prefix="/external_oidc")
-    app.register_blueprint(aggregate.blueprint, url_prefix="/aggregate")
+    if not app.registered:
+        app.register_blueprint(oauth2.blueprint, url_prefix="/oauth2")
+        app.register_blueprint(tokens.blueprint, url_prefix="/token")
+        app.register_blueprint(external_oidc.blueprint, url_prefix="/external_oidc")
+        app.register_blueprint(aggregate.blueprint, url_prefix="/aggregate")
+        app.registered = True
```

Comment:
I think it would be good if we could alternatively just make sure that the setup function is only called once.

So instead of using `@app.before_first_request` to set up, `_setup` could be called from both [wsgi.py](https://github.com/uc-cdis/workspace-token-service/blob/3d9afdc226ba09014274c38cb46fda5fec864316/deployment/uwsgi/wsgi.py#L2) and [run.py](https://github.com/uc-cdis/workspace-token-service/blob/3d9afdc226ba09014274c38cb46fda5fec864316/run.py#L5).

I also think it would be good to rename `_setup` to something like `set_up_app` or `init_app` since it is called from outside this file.